scripts/post-update.sh

```diff
@@ -212,6 +212,20 @@ if ! git remote | grep -q local; then
     git remote add local ~/companion
 fi
 
+#Install isc-dhcp-server
+cd $HOME/companion
+echo "Checking for isc-dhcp-server..."
+if dpkg-query -W -f='${Status} ${Version}' isc-dhcp-server | grep 4.3; then
+    echo "Already installed"
+    sudo dpkg --configure -a
```

Comment:
Why is this necessary ?

Reply:
I faced some issues where, even if the package was installed, it was broken and was not functioning properly, so just for caution, i added this line, that checks if the package is proper even if it is installed.